packages/launcher-component/src/pickers/environmentvars-picker.tsx

```diff
@@ -7,17 +7,17 @@ import { InputProps, Picker } from '../core/types';
 const VALID_ENV_KEY_REGEXP = new RegExp('^$|^[-._a-zA-Z][-._a-zA-Z0-9]*$');
 
 export interface EnvironmentVarsPickerValue {
-  envVars?: Array<{ key: string; value: string }>;
+  envVars?: string[][];
 }
 
 interface EnvironmentVarsPickerProps extends InputProps<EnvironmentVarsPickerValue> {
 }
 
-const NEW_ENTRY = {key: '', value: ''};
+const NEW_ENTRY = ['', ''];
 
 export const EnvironmentVarsPicker: Picker<EnvironmentVarsPickerProps, EnvironmentVarsPickerValue> = {
   checkCompletion: value => !!value.envVars
-    && value.envVars.filter(entry => !VALID_ENV_KEY_REGEXP.test(entry.key)).length === 0,
+    && value.envVars.filter(entry => !VALID_ENV_KEY_REGEXP.test(entry[0])).length === 0,
   Element: props => {
     const entries = props.value.envVars || [NEW_ENTRY];
```

Comment:
Here you need this:
```
const entries = prop.value.envVars && props.value.envVars.length > 0 ? props.value.envVars: [NEW_ENTRY];
```

Because:
- `_.toPairs(undefined)` => `[]`
- `_.toPairs({})` =>  `[]`
- `Boolean([])` => true

Reply:
```suggestion
    const entries = prop.value.envVars && props.value.envVars.length > 0 ? props.value.envVars: [NEW_ENTRY];
```